lib/screens/v2/widget_instance/departures.ex

```diff
@@ -280,18 +286,26 @@ defmodule Screens.V2.WidgetInstance.Departures do
       {ref, [notice]} when is_reference(ref) ->
         {:notice, notice}
 
-      {_key, departure_group} ->
-        {:normal, Enum.take(departure_group, max_entries_per_group)}
+      {_key, [first_departure | _] = departure_group} ->
+        departures =
+          if first_departure |> Departure.time() |> DateTime.diff(now, :minute) <= 2 do
```

Comment:
Unless I'm missing something, this will reduce audio readouts from 2 departures per route to 1 (in the typical case) on _all_ screen types that use the shared departures widget, including bus shelters and e-ink. Admittedly the requested changes in Asana don't specifically say "only on Sectionals" but that might have been the implicit context. I can definitely imagine that in some cases we'd want to keep the current "always 2" readouts — GL e-ink never even displays more than two departures, so the readouts are already quite short and the extra information is probably appreciated. Maybe we should ask Dave to be sure.